packages/eslint-plugin/docs/rules/README.md

```diff
@@ -6,18 +6,36 @@ pagination_prev: null
 slug: /
 ---
 
-`@typescript-eslint/eslint-plugin` includes over 100 rules that detect best practice violations, bugs, and/or stylistic issues specifically for TypeScript code.
-See [Configs](/linting/configs) for how to enable recommended rules using configs.
+`@typescript-eslint/eslint-plugin` includes over 100 rules that detect best practice violations, bugs, and/or stylistic issues specifically for TypeScript code. All of our rules are listed below.
 
-## Supported Rules
+:::tip
+Instead of enabling rules one by one, we recommend using one of [our pre-defined configs](/linting/configs) to enable a large set of recommended rules.
+:::
+
+## Rules
+
+The rules are listed in alphabetical order. You can optionally filter them based on these categories:
 
 import RulesTable from "@site/src/components/RulesTable";
 
-<RulesTable ruleset="supported-rules" />
+<RulesTable />
+
+## Filtering
+
+### Config Group (⚙️)
 
-## Extension Rules
```

Comment:
I just realized https://typescript-eslint.io/rules/#extension-rules is probably the only canonical link to an explanation of extension rules out there. So we probably don't want to get rid of the heading. But I've pestered you so much on this PR 😅 I'll just add it in, and if you @Josh-Cena / @Zamiell disagree we can always revert before the release on Monday.

Reply:
do we really need a separate heading though? it is fairly self-explanatory what an extension rule is.
like, for example, consider `class-methods-use-this`.
if you go to the page for it, it says:
> This rule extends the base [eslint/class-methods-use-this](https://eslint.org/docs/rules/class-methods-use-this) rule. It adds support for ignoring override methods or methods on classes that implement an interface.
After reading this, i am not left wondering, "I have more questions about what an extension rule is, I wonder if there is a more formal definition out there somewhere?"

Reply:
I think folks need somewhere to hard-link to. It's similar to ESLint's issue for fixers vs suggestions: even if they're fairly straightforward for many, some folks coming in fresh have a hard time with them. I personally had a hard time both with fixers-vs-suggestions _and_ what extension rules are.

Reply:
Ooh and - pretty soon I think we'll be able to trim down the metadata list down to one sentence per item!

* Suggestions: can link to https://github.com/eslint/eslint/pull/17657 once it's in
* Formatting rules: https://eslint.style is coming soon

...which just leaves the deeper explanation on extension rules.